database/dml/postgres/step.go

```diff
@@ -29,6 +29,14 @@ OFFSET $3;
 SELECT count(*) as count
 FROM steps
 WHERE build_id = $1
+`
+
+	// SelectStepImagesCount represents a query to select
+	// the count of an images appearances in the database.
+	SelectStepImagesCount = `
+SELECT image, COUNT(image) as count
```

Comment:
Do you think we should lowercase the `COUNT` here similar to how we do in our other count queries?

i.e. https://github.com/go-vela/server/pull/52/files#diff-14ef7dccc93c0e8ae7848e5f5bdc273dR29